components/molecule/inputTags/src/index.js

```diff
@@ -120,15 +134,26 @@ MoleculeInputTags.propTypes = {
   onChange: PropTypes.func,
 
   /* object generated w/ Reacte.createRef method to get a DOM reference of internal input */
-  innerRefInput: PropTypes.object
+  innerRefInput: PropTypes.object,
+
+  /* text to be displayed if there is no tags and the input is empty */
+  placeholder: PropTypes.string,
+
+  /* number of maximum tags that can be added, after reaching this number the component will be disabled */
+  maxTags: PropTypes.number,
+
+  /* prop to indicate that the field is disable (will not render the input) */
+  disabled: PropTypes.bool
 }
 
 MoleculeInputTags.defaultProps = {
   size: inputSizes.MEDIUM,
   value: '',
   tags: [],
   onChangeTags: () => {},
-  onChange: () => {}
+  onChange: () => {},
+  placeholder: '',
+  disabled: false
 }
```

Comment:
Can we remote this? `defaultProps` are not necessary because they can be added on creating the function (line 21).